generators/java/sdk/src/main/java/com/fern/java/client/generators/WrappedRequestGenerator.java

```diff
@@ -115,6 +143,7 @@ public GeneratedWrappedRequest generateFile() {
                 .addAllExtends(extendedInterfaces)
                 .addAllProperties(headerObjectProperties)
                 .addAllProperties(queryParameterObjectProperties)
+                .addAllProperties(pathParameterObjectProperties)
```

Comment:
nit: If this controls the order, we might want to put the path parameter properties first.

Reply:
done